citation/export_data.py

```diff
@@ -10,7 +13,7 @@ def get_attribute_values(attributes):
 
 
 # creates a list of True/False depending whether values are within the items or not - for creating Matrix
-def create_output(items, values):
+def generate_list(items, values):
```

Comment:
I think you can come up with an even better name than this that aids understanding :+1:  :paintbrush: